hathor/cli/run_node.py

```diff
@@ -103,6 +103,8 @@ def create_parser(self) -> ArgumentParser:
         parser.add_argument('--x-rocksdb-indexes', action='store_true', help='Use RocksDB indexes (currently opt-in)')
         parser.add_argument('--x-enable-event-queue', action='store_true', help='Enable event queue mechanism')
         parser.add_argument('--x-retain-events', action='store_true', help='Retain all events in the local database')
+        parser.add_argument('--peer-id-blacklist', action='extend', default=[], nargs='+', type=str,
```

Comment:
I feel you can add an example of use in the PR description, both using cmd line and envvars.

Reply:
Yeah, I think using `HATHOR_PEER_ID_BLACKLIST` could be tricky because it's an `action='extend'` argument. I'm not sure how this is dealt with by `configargparse`.

Reply:
The examples were added in the PR description